docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,70 @@
+---
+id: v1.1.0
+title: v1.1.0
+description: Backstage Release v1.1.0
+---
+
+These are the release notes for the v1.1.0 release of [Backstage](https://backstage.io/).
+
+A huge thanks to the whole team of maintainers and contributors as well as the amazing Backstage Community for the hard work in getting this release developed and done.
+
+## Highlights
+
+### Auth
+
+The auth [backend](https://github.com/backstage/backstage/tree/master/plugins/auth-backend) and [node package](https://github.com/backstage/backstage/tree/master/plugins/auth-node) received a few **BREAKING** changes.
```

Comment:
maybe `[backend] and [node] packages` (one word in second box, and plural packages)